docs/docs/product-documentation.md

```diff
@@ -75,53 +71,48 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
 
 Strategic fit:  
 Cryptocurrency exchange providers like Binance and StatCoin offer exchange of hundreds of different cryptocurrencies. Moreover, these providers store and manage their users' public and private keys. 
-However, Swapchain's unique selling point consists of the fact that it does not save any private or public keys of the users. Also counterparty risks are minimized as the users themselves have to manage their keys necessary for transactions. Thus, no trust towards the trading partner is needed and the transaction fees are significantly lower compared to its competitors.
+However, Swapchain's unique selling point consists of the fact that it does not save any private or public keys of the users as any information stays in the browser and is not sent to any database. Also counterparty risks are minimized as the users themselves have to manage their secret hashes necessary for transactions. Thus, no trust towards the trading partner is needed and the transaction fees are significantly lower compared to its competitors.
 
 #### 2.1.4. Assumptions
 
-It is assumed that future users of the Swapchain web application have a good understanding of blockchains and cryptocurrencies. The reason being that users are self-responsible for their private and public keys. Moreover assets are going to be exchanged and transaction fees may be applied.
+It is assumed that future users of the Swapchain web application have a good understanding of blockchains and cryptocurrencies. The reason being that users are self-responsible for their private and public keys. Moreover assets are going to be exchanged and transaction fees are applied. 
 
 #### 2.1.5. User stories
 
 As a user of Swapchain, I want to exchange BTC for BTS or vice versa without trusting the other party. Also I do not want my personal or private keys to be stored by the system and I do not want to rely on a third party (Zipkin, 2020 [2]).
 
 #### 2.1.6. User interaction and design
 
-The system communicates with the user via a user interface (UI). React.js will be used to create a basic frontend. Moreover, a database will be used as the orderbook.
-Design explorations and wireframes will be added as soon as they are finalised for the Swapchain application.
+The system communicates with the user via a user interface (UI). React.js will be used to create a basic frontend.
```

Comment:
Not sure as we are using .tsx files for React if we can say React.js anyways

Reply:
yes please use React.js